content/PIV/Guides/PIV_Walk-Through.adoc

```diff
@@ -0,0 +1,101 @@
+== PIV Walk-Through
+
+=== Getting Started: SSH Authentication with a YubiKey as a Smart Card
+Build an app that enables your users to register and authenticate with PIV (Personal Identity Verification or Privilege and Identification). This walk-through describes how to integrate the PIV protocol with your application or framework.
+
+PIV enables RSA or ECC sign/encrypt operations using a private key stored on a smart card, through common interfaces such as PKCS#11. The YubiKey 4 and the YubiKey 5 support not only RSA keys, but also link:https://en.wikipedia.org/wiki/Elliptic_Curve_Digital_Signature_Algorithm[Elliptic Curve Digital Signature Algorithm] (ECDSA) keys. Their "touch-policy=always" feature ensures that in addition to entering the PIN, the end-user must touch or tap the Yubikey when prompted. User consent is thus obtained without an intermediary keyboard.
+
+=== About SSH
+SSH (Secure Shell) is a protocol and a software package that enables secure system administration and file transfers over insecure networks. For more information, see link:https://www.ssh.com/ssh/[the SSH Home Page].
+
+=== About the Yubico PIV Tool
+Use this tool for managing the PIV application on a YubiKey. For more information, see link:https://developers.yubico.com/yubico-piv-tool/[the Yubico PIV Tool page].
+
+The Yubico PIV tool is available link:https://developers.yubico.com/yubico-piv-tool/Releases/[on Yubico's release page for it] or on link:https://github.com/Yubico/yubico-piv-tool/releases[GitHub].
```

Comment:
If you are running Linux, you can also install it through `apt` (i.e. by running `apt install yubico-piv-tool`). You decide whether it is necessary to add that or not :-)